app/.server/utils/password.test.ts

```
@@ -0,0 +1,58 @@
+import { afterEach, beforeEach, describe, expect, it } from 'vitest';
+import { comparePassword, getSaltedPassword, hashPassword } from './password';
+
+const plainPasswd =
+  'bf73164d9941ba1afcf8027913aef0631fc001a2170b76746027a552ece3c0e3a665ccc61f2f0bfbc60669bbdf9ce7f560493bc0f1c494e6eddcf03f3c0b53ac'; // an random SHA512 hash
+
+describe('hashPassword()', async () => {
+  let hash1: string | undefined, hash2: string | undefined;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)